zio/bzngio/seeker.go

```diff
@@ -0,0 +1,27 @@
+package bzngio
+
+import (
+	"io"
+
+	"github.com/brimsec/zq/zng/resolver"
+)
+
+type Seeker struct {
+	Reader
+	seeker io.ReadSeeker
+}
+
+func NewSeeker(s io.ReadSeeker, zctx *resolver.Context, framesize int) *Seeker {
```

Comment:
why does the caller need to pass a buffer size here?  that's a change from how existing readers work.

Reply:
I will add NewSeekerWithSize().   Zdx wants to control the read buffer size.